### PR TITLE
list with form: Improved form data handling

### PR DESCRIPTION
**Pull request by groovedk**
Separation of form data and original data

### CODE REVIEW

File: pages/exampleListWithForm/FormSet.tsx [hunk 1]
```diff
@@ -67,9 +62,9 @@ function FormSet({}: Props) {
     }
   }, [saveRequestValue, form]);
 
-  React.useEffect(() => {
-    form.resetFields();
-  }, [form, listSelectedRowKey]);
+  // React.useEffect(() => {
```
Comment (thomasJang) [inline pages/exampleListWithForm/FormSet.tsx:65]: PLZ remove comments

File: pages/exampleListWithForm/useExampleListWithFormStore.ts [hunk 2]
```diff
@@ -207,20 +221,21 @@ export const unSubscribeExampleListWithFormStore = useExampleListWithFormStore.s
...
-  ([listSelectedRowKey]) => {
-    if (listSelectedRowKey) {
-      useExampleListWithFormStore.getState().callDetailApi();
-    } else {
-      // clear
-    }
-  },
-  { equalityFn: shallow }
-);
+// useExampleListWithFormStore.subscribe(
```
Comment (thomasJang) [inline pages/exampleListWithForm/useExampleListWithFormStore.ts:231]: PLZ remove comments

**Review by thomasJang — CHANGES_REQUESTED** [review]
다시하세요.

File: pages/exampleListWithForm/useExampleListWithFormStore.ts (hunk 2 above)
Comment (groovedk) [reply pages/exampleListWithForm/useExampleListWithFormStore.ts]: complete

File: pages/exampleListWithForm/FormSet.tsx (hunk 1 above)
Comment (groovedk) [reply pages/exampleListWithForm/FormSet.tsx]: complete

**Pull request merged**